forge-gui/res/cardsfolder/upcoming/seething_skitter_priest.txt

```diff
@@ -0,0 +1,14 @@
+Name:Seething Skitter-Priest
+ManaCost:2 W B
+Types:Creature Phyrexian Cleric
+PT:3/3
+K:Toxic:1
+T:Mode$ ChangesZone | ValidCard$ Card.Self | Origin$ Any | Destination$ Battlefield | Execute$ TrigEffect | TriggerDescription$ When CARDNAME enters the battlefield, creatures you control and creature cards in your hand perpetually gain "When this creature dies, create a 1/1 colorless Phyrexian Mite artifact creature token with toxic 1 and "This creature can't block.""
+SVar:TrigEffect:DB$ Effect | RememberObjects$ Valid Creature.YouCtrl,ValidHand Creature.YouOwn | StaticAbilities$ PerpetualTokenTrig | Name$ Seething Skitter-Priest's Perpetual Effect | Duration$ Permanent
+SVar:PerpetualTokenTrig:Mode$ Continuous | Affected$ Card.IsRemembered | AddTrigger$ ETBToken | EffectZone$ Command | AffectedZone$ Battlefield,Hand,Graveyard,Exile,Stack,Library,Command | Description$ This creature perpetually gains "When this creature dies, create a 1/1 colorless Phyrexian Mite artifact creature token with toxic 1 and "This creature can't block.""
```

Comment:
Not that it doesn't work but the Trigger name being ETB doesn't make sense when the trigger is "When this creature dies"

You can ignore this if you want to but if you change it remember line 9 too